rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_fw_dynamics_plugin.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2017 Pavel Vechersky, ASL, ETH Zurich, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROTORS_GAZEBO_PLUGINS_FW_DYNAMICS_PLUGIN_H
+#define ROTORS_GAZEBO_PLUGINS_FW_DYNAMICS_PLUGIN_H
+
+#include <gazebo/common/Plugin.hh>
+#include <gazebo/common/common.hh>
```

Comment:
I spotted this recent commit: #351 and followed the same order ... If the change in this commit was intentional it would suggest the capitalized files are grouped together vs non-capitilized. However, this is not consistent across all the current plugins, so hard to say.